include/seqan3/range/views/pairwise_combine.hpp

```diff
@@ -446,7 +59,7 @@ class pairwise_combine_view : public std::ranges::view_interface<pairwise_combin
     using const_iterator    = transformation_trait_or_t<std::type_identity<iterator_type<underlying_range_type const>>,
                                                         void>;
```

Comment:
I know you didn't touch this line, but if you delete the spaces, everything will fit on one line.
```suggestion
    using const_iterator = transformation_trait_or_t<std::type_identity<iterator_type<underlying_range_type const>>, void>;
```

Reply:
I think it is one character too long. I split it. :)